CMEW/app/configure_standardise/bin/configure_standardise.sh

```diff
@@ -3,14 +3,94 @@
 # The LICENSE.md file contains full licensing details.
 # Send the output from 'set -x' to 'stdout' rather than 'stderr'.
 BASH_XTRACEFD=1
-set -eux
+set -euo pipefail
+set -x
 
-echo "Running configure_standardise"
+echo "[INFO] Running configure_standardise for REF and EVAL runs"
 
-# Create request configuration file and variables file.
-cmew-esmvaltool-env create_request_file.py
+# ---------------------------------------------------------------------------
+# 0. Derive per-run request paths if not provided by Cylc
+# ---------------------------------------------------------------------------
+# We still honour REQUEST_PATH_REF / REQUEST_PATH_EVAL if flow.cylc sets them,
+# but we can safely derive them here from CYLC_WORKFLOW_SHARE_DIR as a fallback.
+
+: "${CYLC_WORKFLOW_SHARE_DIR:?CYLC_WORKFLOW_SHARE_DIR must be set}"
+
+REQUEST_PATH_REF="${REQUEST_PATH_REF:-${CYLC_WORKFLOW_SHARE_DIR}/etc/request_ref.cfg}"
+REQUEST_PATH_EVAL="${REQUEST_PATH_EVAL:-${CYLC_WORKFLOW_SHARE_DIR}/etc/request_eval.cfg}"
+
```

Comment:
The user will not be modifying `flow.cylc` so I don't think there is a need to derive them here. My previous experience with workflows is that it can be confusing to have more than one place that the value could be defined.

So I'd recommend removing the derivation. You could instead include a check-and-fail that `REQUEST_PATH_*` have been defined, as in lines 26-31 for the model variables.